src/rulesets/jsonapi.ts

```diff
@@ -21,7 +21,7 @@ function loadSchemaFromFile(filename) {
 
 export const rules = {
   statusCodes: ({ operations }: SnykApiCheckDsl) => {
-    operations.requirement.must(
+    operations.requirementOnChange.must(
```

Comment:
this is probably fine. It might ask people to add status codes they don't currently support though when they go change those endpoints. 

If we only want to enforce this on new surface area use `added`